xwiki-platform-core/xwiki-platform-realtime/xwiki-platform-realtime-api/src/main/resources/META-INF/components.txt

```diff
@@ -0,0 +1,2 @@
+500:org.xwiki.realtime.internal.XWikiRealtimeDocumentLockEditConfirmationChecker
+org.xwiki.realtime.internal.DefaultRealtimeEditorManager
```

Comment:
I usually list component implementations in alphabetical order (helpful when there are many of them), but it's probably another unwritten rule :)